site/plugin/cni-plugin-how-to.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Using the Weave Net CNI Plugin
+layout: default
+---
+
+CNI, the [_Container Network Interface_](https://github.com/appc/cni#cni---the-container-network-interface),
+is a proposed standard for configuring network interfaces for Linux
+application containers.  CNI is supported by
+[Kubernetes](http://kubernetes.io/), an open source container cluster
+manager built by Google.
+
+###Installing the Weave Net CNI plugin
+
+The Weave Net CNI plugin is installed when you run `weave setup`, if
+it your machine has the directories normally used to host CNI plugins.
+To create those directories, run (as root):
+
+    mkdir -p /opt/cni/bin
+    mkdir -p /etc/cni/net.d
+
+Then run:
+
+    weave setup
+
+###Launching Weave Net
+
+To create a network which can span multiple hosts, the Weave peers must be connected to each other, by specifying the other hosts during `weave launch` or via
+[`weave connect`](/site/using-weave/finding-adding-hosts-dynamically.md).
+
+See [Deploying Applications to Weave Net](/site/using-weave/deploying-applications.md#peer-connections) for a discussion on peer connections. 
+
+As well as launching Weave Net, you have to run an extra command to
+configure the Weave bridge:
```

Comment:
We don't want to give the impression that `weave expose` generally configures the weave bridge. So... "... run an extra command to perform some additional configuration of the Weave bridge...".